components/webext-storage/src/webext-storage.udl

```diff
@@ -42,9 +53,62 @@ interface WebExtStorageStore {
     [Throws=WebExtStorageApiError]
     JsonValue get([ByRef] string ext_id, JsonValue keys);
 
+    [Throws=WebExtStorageApiError]
+    u64 get_bytes_in_use([ByRef] string ext_id, JsonValue keys);
+
     [Throws=WebExtStorageApiError]
     StorageChanges remove([ByRef] string ext_id, JsonValue keys);
 
     [Throws=WebExtStorageApiError]
     StorageChanges clear([ByRef] string ext_id);
+
+    [Self=ByArc]
+    WebExtStorageBridgedEngine bridged_engine();
+
+    [Throws=WebExtStorageApiError]
+    sequence<SyncedExtensionChange> get_synced_changes();
+};
+
+// Note the canonical docs for this are in https://searchfox.org/mozilla-central/source/services/interfaces/mozIBridgedSyncEngine.idl
```

Comment:
I don't think that's true, and that I expect we should be able to kill that idl after this landed. The canonical docs for this new world order are at https://github.com/mozilla/application-services/blob/main/components/sync15/src/engine/bridged_engine.rs